components/mongodb/actions/find-document-by-id/find-document-by-id.mjs

```diff
@@ -0,0 +1,35 @@
+import mongodbApp from "../../mongodb.app.mjs";
+
+export default {
+  key: "mongodb-find-document-by-id",
+  name: "Find Document by Id",
+  description: "Retrieves a single document by ID. [See the docs here](https://docs.mongodb.com/manual/reference/method/db.collection.find/)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    mongodbApp,
+    collection: {
+      propDefinition: [
+        mongodbApp,
+        "collection",
+      ],
+    },
```

Comment:
The description for this is "Collection where the new document will be inserted". It should be something like "Collection containing the document to find."